### PR TITLE
Disable graph transforms library in windows pip packages.

### PR DESCRIPTION
**Pull request by gunan**

### CODE REVIEW

**Review by petewarden — APPROVED** [review]

File: tensorflow/python/BUILD [hunk 1]
```diff
@@ -2311,10 +2311,11 @@ tf_py_wrap_cc(
         "//tensorflow/core:lib",
         "//tensorflow/core/debug",
         "//tensorflow/core/distributed_runtime:server_lib",
-        "//tensorflow/tools/graph_transforms:transform_graph_lib",
         "//tensorflow/tools/tfprof/internal:print_model_analysis",
         "//util/python:python_headers",
-    ] + tf_additional_lib_deps() + tf_additional_plugin_deps(),
+    ] + if_not_windows([
+        "//tensorflow/tools/graph_transforms:transform_graph_lib",
+    ] + tf_additional_lib_deps() + tf_additional_plugin_deps()
```
Comment (mrry) [inline tensorflow/python/BUILD]: Missing closing parenthesis for the `if_not_windows()` call?

**Review by mrry — APPROVED** [review]

**Comment by gunan** [comment]
Jenkins, test this please.

**Comment by gunan** [comment]
http://ci.tensorflow.org/job/tf-pr-win-bzl/1/ running bazel windows tests.

**Comment by gunan** [comment]
Build failed with:
http://ci.tensorflow.org/job/tf-pr-win-bzl/1/console

Will also do the same conditional include swig includes.

**Comment by gunan** [comment]
Jenkins, test this please.

**Comment by gunan** [comment]
all tests passed (android was failing before, now that is also passing.

**Pull request merged**